pkg/compute/compute_integration_test.go

```diff
@@ -469,6 +483,93 @@ func TestBuild(t *testing.T) {
 	}
 }
 
+func TestBuildAssemblyScript(t *testing.T) {
+	if os.Getenv("TEST_COMPUTE_BUILD_ASSEMBLYSCRIPT") == "" && os.Getenv("TEST_COMPUTE_BUILD") == "" {
```

Comment:
Personally, I think it would be kind of annoying to always minimally have to supply two environment variables for testing.

I definitely don't think we want to be string comparing some value in `TEST_COMPUTE_BUILD`, but rather using the other language-specific boolean envs and dropping `TEST_COMPUTE_BUILD` since that would be implied.

Reply:
I also don't really have the context for what either of these do, so I could be overlooking something.

Reply:
>Personally, I think it would be kind of annoying to always minimally have to supply two environment variables for testing.

The logic here may be confusing. You only need to supply `TEST_COMPUTE_BUILD` if you want to run all of the build tests i.e. rust and assemblyscript. Which is what is already happening in CI.

This logic skips the test if both `TEST_COMPUTE_BUILD_RUST` and `TEST_COMPUTE_BUILD` are set. i.e. it gives us the flexibility to only run a single toolchain test or all of them.